src/test/java/testng/smoke/AddNewWidgetTest.java

```diff
@@ -0,0 +1,64 @@
+package testng.smoke;
+
+import com.report.portal.business.constants.WidgetType;
+import com.report.portal.business.steps.BaseSteps;
+import com.report.portal.business.steps.DashboardsSteps;
+import io.qameta.allure.Description;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static com.codeborne.selenide.Selenide.closeWebDriver;
+import static com.report.portal.business.constants.WidgetType.*;
+import static com.report.portal.business.steps.DashboardsSteps.createWidgetName;
+import static com.report.portal.core.utils.data.RandomSequenceCreator.generateRandomInt;
+
+public class AddNewWidgetTest {
+
+    BaseSteps baseSteps = new BaseSteps();
```

Comment:
Could be private.
Here and in other places

Reply:
Fixed!